src/models/user.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from 'zod';
+
+export const FlowSchema = z.object({
+    ty: z.literal('flow'),
```

Comment:
Any reason not to use the unabbreviated `type` instead of `ty`?

Reply:
same concern with `ts`.

Reply:
That's because `type` is a keyword in TypeScript. The `ty` identifier name is actually a common abbreviation. We even see it in the source code for `rustc`.